viscy/trainer.py

```diff
@@ -128,3 +133,240 @@ def precompute(
             include_wells=include_wells,
             exclude_fovs=exclude_fovs,
         )
+
+    def compute_dynacell_metrics(
+        self,
+        target_database: Path,
+        pred_database: Path,
+        output_dir: Path,
+        method: str = "intensity",
```

Comment:
Can this be all supported literals rather than a free string? Then lightning CLI can check the input type.

Reply:
yes, we will update this once we have all the possible ones. We might make it it's own list for the organelles. thanks